sql-statements/sql-statement-restore.md

```diff
@@ -0,0 +1,161 @@
+---
+title: RESTORE
+summary: TiDB 数据库中 RESTORE 的使用概况。
+category: reference
+---
+
+# RESTORE
+
+`RESTORE` 语句用于执行分布式恢复，把 [`BACKUP` 语句](/sql-statements/sql-statement-backup.md)生成的备份文件恢复到 TiDB 集群中。
+
+`RESTORE` 语句使用的引擎与 [BR](/br/backup-and-restore-use-cases.md) 相同，但恢复过程是由 TiDB 本身驱动，而非单独的 BR 工具。BR 工具的优势和警告也适用于 `BACKUP` 语句。需要注意的是，**`RESTORE` 语句目前不遵循 ACID 原则**。
+
+执行 `RESTORE` 语句前，确保集群已满足以下要求：
+
+* 集群处于“下线”状态，当前的 TiDB 会话是唯一在访问要恢复的表的活跃 SQL 连接。
+* 执行全量恢复时，确保即将恢复的表不存在于集群中，因为现有的数据可能被覆盖，从而导致数据与索引不一致。
+* 执行增量恢复时，表的状态应该与创建备份时 `LAST_BACKUP` 时间戳的状态完全一致。
+
+执行 `RESTORE` 需要 `SUPER` 权限。此外，执行恢复操作的 TiDB 节点和集群中的所有 TiKV 节点都必须有对目标存储的读权限。
+
+`RESTORE` 语句开始执行后将会被封锁，直到整个恢复任务完成、失败或取消。因此，执行 `RESTORE` 时需要准备一个持久的连接。如需取消任务，可执行 [`KILL TIDB QUERY`](/sql-statements/sql-statement-kill.md) 语句。
+
+一次只能执行一个 `BACKUP` 和 `RESTORE` 任务。如果 TiDB server 上已经在执行一个 `BACKUP` 或 `RESTORE` 语句，新的 `RESTORE` 将等待前面所有的任务完成后再执行。
+
+`RESTORE` 只能在 "tikv" 存储引擎上使用，如果使用 "mocktikv" 存储引擎， `RESTORE` 操作会失败。
+
+## 语法图
+
+**RestoreStmt:**
+
+![RestoreStmt](/media/sqlgram/RestoreStmt.png)
+
+**BRIETables:**
+
+![BRIETables](/media/sqlgram/BRIETables.png)
+
+**RestoreOption:**
+
+![RestoreOption](/media/sqlgram/RestoreOption.png)
+
+**Boolean:**
+
+![Boolean](/media/sqlgram/Boolean.png)
```

Comment:
This image looks different from the one in https://github.com/pingcap/docs/pull/2618. 
![image](https://user-images.githubusercontent.com/41534398/87053090-1f73eb80-c234-11ea-997d-1b527303d79a.png)